QandA.md

```diff
@@ -59,39 +59,39 @@ Für manche fängt Non-24 im frühen Kindesalter an. Für andere beginnt es in d
+Für jene, bei welchen eine Behandlung unwirksam ist, kann es Non-24 unmöglich machen, eine feste Arbeits- oder Schulzeit einzuhalten. Für diejenigen, die Berufe mit festen Arbeitszeiten haben, sowie für Schüler und Studenten, die Unterricht zu gewöhnlichen Zeiten besuchen müssen, ist Non-24, falls die Behandlung fehlschlägt, – eine Behinderung. Schulen und Arbeitgeber in den USA sind laut dem Americans with Disabilities Act verpflichtet, angemessene Vorkehrungen zu ermöglichen, einschließlich Teilzeitarbeit oder angepasste Arbeitszeiten.
 
-An dieser Stelle enthält der englische Originaltext Informationen welche sich auf die US-amerikanische Gesetzgebung zum Thema Behinderungen beziehen und wurden daher weggelassen. Andere Länder mögen ähnliche Anforderungen an Vorkehrmaßnahmen stellen.
+An dieser Stelle enthält der englische Originaltext Informationen, welche sich auf die US-amerikanische Gesetzgebung zum Thema Behinderungen beziehen und wurden daher weggelassen. Andere Länder mögen ähnliche Anforderungen an Vorkehrungen stellen.
 
 ### Welche Vorkehrungen würden helfen?
 
-Eine größere Flexibilität bei den Arbeitszeiten. Es Arbeitnehmern ermöglichen von zu Hause aus zu arbeiten, zu Zeiten an denen sie können. Kürzere Arbeitszeiten mit entsprechend weniger Entgelt erlauben, für diejenigen die das wegen chronischer Müdigkeit benötigen. Es Schüler und Studenten zu erlauben, Unterrichtsstunden, die zu nicht gewöhnlichen Zeiten stattfinden, aufgezeichnet oder online verfügbar sind, zu besuchen, oder anzuerkennen, dass sie einige Stunden versäumen werden aber trotzdem mit dem Lernstoff weitermachen können. Dafür sorgen, dass Prüfungen zu angemessenen Zeiten stattfinden. Einen leichteren Lehrplan ermöglichen, auch wenn dies bedeutet den Abschluss zu verzögern.
+Eine größere Flexibilität bei den Arbeitszeiten. Es Arbeitnehmern ermöglichen, von zu Hause aus zu arbeiten, zu Zeiten, zu denen sie können. Kürzere Arbeitszeiten mit entsprechend weniger Entgelt erlauben für diejenigen, die das wegen chronischer Müdigkeit benötigen. Es Schüler und Studenten zu erlauben, Unterrichtsstunden, die zu nicht gewöhnlichen Zeiten stattfinden, aufgezeichnet oder online verfügbar sind, zu besuchen oder anzuerkennen, dass sie einige Stunden versäumen werden aber trotzdem mit dem Lernstoff weitermachen können. Dafür sorgen, dass Prüfungen zu angemessenen Zeiten stattfinden. Einen leichteren Lehrplan ermöglichen, auch wenn dies bedeutet, den Abschluss zu verzögern.
```

Comment:
- added missing commas
- replaced "an" with "zu" (correct when referring to times)